internal/providers/scaleway/scaleway.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Just a small nit, would it be possible to describe how scale way is fetching the data in a comment above. similar to how it was done [here](https://github.com/coreos/ignition/blob/main/internal/providers/hetzner/hetzner.go#L15).